### PR TITLE
Adding Adagrams Project

### PR DESCRIPTION
**Pull request by jbieniosek**

### CODE REVIEW

File: adagrams/intro.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Project: Swap Meet
```
Comment (audreyandoy) [inline adagrams/intro.md:1]:
```suggestion
# Project: Adagrams
```

File: adagrams/submit.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# Submit: Swap Meet
```
Comment (audreyandoy) [inline adagrams/submit.md:1]:
```suggestion
# Submit: Adagrams
```

File: adagrams/submit.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+## Submit Project for Grading
+
+<!-- prettier-ignore-start -->
+### !challenge
+* type: testable-project
+* id: 79Czep
+* title: Fork URL for Grading
+* upstream: https://github.com/AdaGold/swap-meet
```
Comment (audreyandoy) [inline adagrams/submit.md:10]:
```suggestion
* upstream: https://github.com/AdaGold/adagrams-py
```

**Review by audreyandoy — COMMENTED** [review]
Hi! I adjusted titles and links to reflect adagrams, not swap-meet.

File: adagrams/submit.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+<!--optional, checkpoints only-->
+##### !rubric
+
+##### !end-rubric
+
+<!--optional-->
+##### !explanation
+
+##### !end-explanation
```
Comment (jbieniosek) [inline adagrams/submit.md:97]:
```suggestion

```

Comment (jbieniosek) [inline adagrams/submit.md:97]:
```suggestion

```

Comment (jbieniosek) [inline adagrams/submit.md:97]:
```suggestion

```

**Pull request merged**